src/main/kotlin/tech/thatgravyboat/craftify/config/Config.kt

```diff
@@ -189,6 +189,14 @@ object Config : Vigilant(File("./config/craftify.toml")) {
     )
     var announceNewSong = 0
 
+    @Property(
+            type = PropertyType.TEXT,
+            name = "Announcement Message",
+            category = "General",
+            description = "Format for the announcement message (chat message)"
```

Comment:
Provide what the user can input into a message for replacements just incase if they forget, also change the replacements to be `${variable}` instead.

Reply:
Is this what you intended?